### ISSUE TITLE
🐛 Error stack trace lines are offset by 1

### ISSUE THREAD

**Issue by jer3m01** [rome/tools#1380]
## Environment

```
$ rome rage --summary
Environment Variables
  Object {
    LANG: "C.UTF-8"
    TERM: "xterm-256color"
  }

User Config
  unset

Rome Version
  10.0.5-beta-dev

Node Version
  12.18.3

Platform
  linux x64 4.4.0-19041-Microsoft

Terminal Features
  Object {
    background: "unknown"
    colorDepth: 8
    cursor: true
    hyperlinks: true
    isTTY: true
    progress: true
    unicode: true
    columns: OneIndexedNumber<472>
  }

Client Flags
  Object {
    clientName: "cli"
    silent: false
    cwd: AbsoluteFilePath</mnt/d/Webstorm/jer3m01/rome>
    realCwd: AbsoluteFilePath</mnt/d/Webstorm/jer3m01/rome>
  }

Server Status
  Object {
    projects: Array [
      Object {id: 0}
      Object {id: 1}
    ]
    server: Object {
      heapTotal: 29_929_472
      pid: 494
      uptime: 1.0194439
    }
    workers: Array [
      Object {
        astCacheSize: 0
        heapTotal: 29_929_472
        ownedBytes: 0n
        ownedFileCount: 0
        pid: 494
        uptime: 1.0193367
      }
    ]
  }
```

## Steps To Reproduce

1. Throw an error
![image](https://user-images.githubusercontent.com/33844379/109430534-2ab59a80-7a02-11eb-8847-fa82c3bfded8.png)
![image](https://user-images.githubusercontent.com/33844379/109430537-2ee1b800-7a02-11eb-8727-d614c0ad97f1.png)
![image](https://user-images.githubusercontent.com/33844379/109430539-31dca880-7a02-11eb-98de-8ca39bd3f87e.png)

## Expected Results

Lines shouldn't be offset by 1

**Comment by jer3m01** [comment]
Did a bit of digging and have a suspicion the issue is caused by `codec-source-map`.

https://github.com/rome/tools/blob/main/internal/v8/sourceMapManager.ts#L150

**Comment by sebmck** [comment]
I wonder if it's our sourcemaps that are being generated off-by-one, or it's how we're processing the line provided by the V8 API when capturing errors.

**Comment by jer3m01** [comment]
How far have we diverted from <https://github.com/rome/tools/tree/main/internal/codec-source-map>? 
Could check if this occurs there too, if not we can update.

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
